private_sharing/models.py

```diff
@@ -81,12 +81,11 @@ def get_visible_user_projects(member):
     Returns a list of projects where membership is publicly
     visible.
     """
-    projects = []
     project_ids = DataRequestProjectMember.objects.filter(visible=True,
```

Comment:
Rather than the list comprehension and additional database query below, could this database query filter on `project__approved=True`?